src/ProgressOnderwijsUtils/RandomHelper.cs

```diff
@@ -14,6 +15,10 @@ public sealed class RandomHelper
         public static RandomHelper Insecure(int seed)
             => new RandomHelper(new Random(seed).NextBytes);
 
+        [NotNull]
+        public static RandomHelper JmplicitlyInsecure([CallerLineNumber] int linenumber = -1, [CallerFilePath] string filepath = "", [CallerMemberName] string membername = "")
+            => new RandomHelper(new Random((linenumber + filepath + membername).GetHashCode()).NextBytes);

```

Comment:
Gethashcode is helaas niet deterministisch over runs

Reply:
Oh... Dan zelf maar een method verzinnen om van 2 strings + 1 int een int te maken? Of weet jij iets anders handigs?

Reply:
Helaas is ook `HashCode` niet stabiel over runs; tja; in de sql-mapper gebruiken we dit: https://github.com/progressonderwijs/ProgressOnderwijsUtils/blob/8ccb96682cf74555659a776ebf3d31ce7e07b82d/src/ProgressOnderwijsUtils/Data/ParameterizedSqlObjectMapper.cs#L153

Case-insensitivity vind je naturlijk niet boeiend, maar iets super-simpels als dat is al OK.  Een van de secure hashes kan natuurlijk ook; al zijn die een beetje overkill zal het vermoedelijk niet uitmaken voor merkbare perf.

Reply:
Het hoeft ook niet echt een hash-code te zijn, waarschijnlijk is elke functie van (string, string, int) naar int die voldoende gevoelig is voor de invoer en waarbij alle uitkomstwaarden ongeveer even "waarschijnlijk" zijn goed genoeg voor dit doeleinde. Dus heb zelf even iets in elkaar geflanst, kijk maar wat je vindt.

Reply:
Nou ja ik heb even de implementatie van die ```CaseInsensitiveHash``` bekeken, en die doet ongeveer dezelfde soort dingen als ik in mijn method deed, alleen waarschijnlijk op een wat meer beredeneerde/robuuste manier. Dus toch die maar gebruikt.

Reply:
Ik kom hier vandaag niet meer aan toe, maar trek gerust iemand anders erbij voor de review..,

Mee eens dat het niet super robuust hoeft te zijn.